src/types/index.ts

```diff
@@ -28,6 +28,7 @@ type AdsProviderProps = {
   debug?: boolean;
   enableLazyload?: boolean;
   enableRefresh?: boolean;
+  children: any;
```

Comment:
@iamtun would you mind to change `children: any`? 

There's a better way that react exposes to add the `children` typing which is using `React.PropsWithChildren<>`. 

For eg: 

```typescript
type AdsProviderProps = React.PropsWithChildren<{
  ads: AdItem[];
  debug?: boolean;
  enableLazyload?: boolean;
  enableRefresh?: boolean;
}>;
```

If you can make that change I would be grateful, because we ensure validating correctly the `children` for those that are using Typescript.

Reply:
Thanks for your idea. I will change it. After I change that can I recreate pull request for you or can you update it for the new version

Reply:
You're welcome! No need to recreate the PR, just commit to your branch and it will automatically be reflected here. Thank you very much for your efforts!

Reply:
i have change it and recommit. you can review it and give me advice. Thanks

Reply:
Perfect! I'm going to merge it and release a newer version. ASAP I'll let you know when a new version is published!